### DISCUSSION TITLE
Pricing for addtional minutes: Question about the minute multiplier for mac

### DISCUSSION THREAD

**Discussion by Siedlerchr** [community/community#26883] [Actions]
<p>I have read the help on the billing topic<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" target="_blank" rel="noopener nofollow ugc">About billing for GitHub Actions - GitHub Docs</a></h3>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>but I am a bit confused when it comes to addtional minutes.<br>
Let’s say I have used all my free minutes and consider the following:</p>
<p>My workflow takes 10 minute on a mac machine.<br>
And one addtional minute for mac costs $0.08</p>
<p>Would that make 10*0.08 = $0.80 or do I still need to add the multiplier and it woudl cost me 10*10*$0.08 = $8 ?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/siedlerchr">@siedlerchr</a> ,</p>
<p>There is a note in the docs,</p>
<blockquote>
<p><strong>Note:</strong>  Minute multipliers do not apply to the per-minute rates shown below.<br>
</p>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/47067810e3369242080515ac68872cc47ae59176.png" data-download-href="/uploads/short-url/a8jL89PpHYduM1Nb6rYi8XwcHau.png?dl=1" title="note" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/46307996/181100466-eb45417b-66be-4781-bb21-db5c6325bd1e.png" alt="note" data-base62-sha1="a8jL89PpHYduM1Nb6rYi8XwcHau" width="690" height="243" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/47067810e3369242080515ac68872cc47ae59176_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">note</span><span class="informations">942×333 6.15 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
</blockquote>
<p>So, the the cost should be “<strong>10 * $0.08 = $0.80</strong>”.</p>

**Comment by Siedlerchr** [comment]
<p>I have read the note but still was confused. So the minute multiplier only apply to the “free” minutes,</p>
